guarana/logger.py

```diff
@@ -0,0 +1,51 @@
+import logging
```

Comment:
Why do we need this whole configuration stuff? I would be in favor of just using logger.warning, logger.error etc. Since the application itself (braincolis) is already configuring this stuff. WDYT?

Reply:
Because Because the segment uses its own logger, which is where the event information is passed, an interrupt is necessary to send this to stdout through loguru. Braincolis has an interruption too, but the one used there wouldn't get the logs sent at the level that the segment is sending

Reply:
All these configs do not need to be modified by the user, for sdk purposes the user will only import the eventTracker and the logs are ready to go

Reply:
> All these configs do not need to be modified by the user, for sdk purposes the user will only import the eventTracker and the logs are ready to go

yes!

Reply:
My implementation proposal is very simple:

just log something in each of the functions on these sdk https://github.com/clicampo/guarana-py/pull/1/files#diff-c5347857c57978901694129ec6c1ace02637ed4b46da13de48147adbbab7b48aR10 like this one. 

If we need a new interruption let's add that on the current braincolis code that is already doing that https://github.com/clicampo/braincolis/blob/e4fb02c50af8030b1ca87ffb4451536d8e2866af/app/core/logger.py, no need to repeat it.